src/modules/launcher/PowerLauncher/SettingsReader.cs

```diff
@@ -227,9 +227,12 @@ private static IEnumerable<PluginAdditionalOption> CombineAdditionalOptions(IEnu
             var defaultOptions = defaultAdditionalOptions.ToDictionary(x => x.Key);
             foreach (var option in additionalOptions)
             {
-                if (defaultOptions.ContainsKey(option.Key))
+                if (option.Key != null)
                 {
-                    defaultOptions[option.Key].Value = option.Value;
+                    if (defaultOptions.ContainsKey(option.Key))
+                    {
+                        defaultOptions[option.Key].Value = option.Value;
+                    }
                 }
```

Comment:
Suggesting to combile both condition in a single if statement
```suggestion
                if (option.Key != null && defaultOptions.ContainsKey(option.Key))
                {
                    defaultOptions[option.Key].Value = option.Value;
                }
```

Reply:
Done